packages/core/tests/controllers/ConnectorController.test.ts

```diff
@@ -179,7 +179,7 @@ describe('ConnectorController', () => {
         imageId: undefined,
         imageUrl: undefined,
         name: 'Auth',
-        type: 'MULTI_CHAIN',
+        type: 'AUTH',
```

Comment:
Do we test the both multi-chain ones and auth ones? If not can we add the missing one to the tests?